test/components/modal.spec.tsx

```diff
@@ -131,7 +131,7 @@ describe('<Modal />', () => {
         await waitFor(() => expect(onRequestClose).to.have.been.calledWithMatch({source: 'backdrop'}));
     });
 
-    it('calls onRequestClose with source equal to children when the child is clicked', async function() {
+    it('calls onRequestClose with source equal to the data-slot of the clicked element', async function() {
```

Comment:
`data-slot="child"` or `data-slot="children"`? I see inconsistency

Reply:
Now the test is simply checking that the proper data-slot is returned for whatever child element is clicked. There's no longer a wrapper with data-slot equal to children